elide/runtime/js/modules/fs/promises.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Intrinsic: Filesystem (Promises).
+ *
+ * Provides a shim which offers a `fs/promises` module implementation that is compatible with Node.js-style imports.
+ */
+
+export default {
```

Comment:
could probably leave this empty to match all other unimplemented intrinsic modules